docs/examples/frameworks/tensorflow/tensorflow-dataset.ipynb

```diff
@@ -21,62 +21,36 @@
    "metadata": {},
```

Comment:
<p>Can we do anything about those warnings? Feel free to consider this out of scope</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>I tried, but the suggested alternative does not work. I'll look at it separately. </p>